src/board.py

```diff
@@ -0,0 +1,35 @@
+class Board:
+    def __init__(self):
+        self.starter_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+        self.player_one = "X"
+        self.player_two = "O"
+
+    def count_marks_in_board(self, board, player_one, player_two):
+        for mark in board:
+            total_marks_on_board = board.count(player_one) + board.count(player_two)
+        return total_marks_on_board
+
+    def get_current_player(self, total_marks_on_board):
+        if total_marks_on_board % 2 == 0:
+            return self.player_one
+        else:
+            return self.player_two
+
+    def assign_board_index_to_current_player_mark(
+        self, user_input, board, total_marks_on_board
+    ):
+        input_index = user_input - 1
+        board[input_index] = self.get_current_player(total_marks_on_board)
+        return board[input_index]
+
+    def mark_board_with_user_selection(self, user_input, board, total_marks_on_board):
+        self.assign_board_index_to_current_player_mark(
+            user_input, board, total_marks_on_board
+        )
+        return board
+
+    def is_board_full(self, total_marks_on_board, board):
+        if total_marks_on_board == len(board):
+            return True
+        else:
+            return False
```

Comment:
I see some methods in this class that seem to denote actions, but I think of a board as more of a static object. Boards don't have a lot of behavior, but players do. Do you see anything in here that might be appropriate to extract into a player class?

Another note on naming: generally wouldn't need to include the class name as a part of method names. So something like `is_board_full` would be called as `board.is_board_full`, which reads a little funny. This can also call out when another class might be appropriate. For instance, if you write a method with a name that indicates it takes an action on the board, that calls out that it's kind of funny to think of a board taking an action on itself.